webrtc_to_hls/README.md

```diff
@@ -10,7 +10,13 @@ This demo is responsible for:
 ### Mac OS X
 
 ```
-brew install srtp libnice clang-format ffmpeg opus fdk-aac
```

Comment:
I assume that libnice and clang-format are no longer needed

Reply:
Yup